TSQL_Parser/TSQL_Parser/Clauses/Parsers/TSQLMergeIntoClauseParser.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+using TSQL.Tokens;
+
+namespace TSQL.Clauses.Parsers
+{
+	internal class TSQLMergeIntoClauseParser

```

Comment:
What caused you to need a MERGE specific INTO clause parser instead of using the existing one?

Reply:
I see that the need was to allow the "AS table_alias" within the clause parsing. This also means that we'll probably eventually have to add a subclass of TSQLIntoClause that exposes the AS.